frontend-site/src/views/OfficePolicy.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <v-container grid-list-lg>
+    <v-layout row v-for="(policy, i) in policies" :key="i">
+      <v-flex>
+        <v-card>
+          <v-card-title primary-title>
+            <h1 class="title">{{ policy.name }}</h1>
+          </v-card-title>
+          <v-card-text>
+            <span
+              v-for="(entry, j) in policy.rules"
+              :key="j"
+              class="d-block"
+              v-text="entry"/>
+          </v-card-text>
+        </v-card>
+      </v-flex>
+    </v-layout>
+    <v-layout row>
+      <v-flex>
+        <v-card>
+          <v-card-title primary-title>
+            <h1 class="title">Space Policy</h1>
+          </v-card-title>
+          <v-card-text>
+            <span class="d-block">
+              The LUG office is a shared resource and thus, we have many people coming in any time during the day. While you are free to use the desks and chairs of the ACM, you may not 'hog' a seat. You must give up your seat to another person that wants to do any of the following:
```

Comment:
good, just if on line 27 here you could change ACM to LUG. Sorry I missed that when I copy and pasted it

Reply:
Would the same apply to line 36 as well?

Reply:
Made changes to both lines. Let me know if you want me to change that second line back to ACM

Reply:
yes both lines I didn't read it closely enough thanks for catching that